openshift-install-powervs

```diff
@@ -1288,28 +1348,16 @@ function main {
       shift
       var="$1"
       #TODO: Need validation on variable key=value, currently it is only checking if equals sign is present or not
-      [[ "$var" != *"="* ]] && error "The given -var option must be a variable name and value separated by an equals sign, eg: -var \"key=value\""
+      [[ "$var" != *"="* ]] && error "The given -var option must be a non-empty variable name and non-empty value separated by an equals sign, eg: -var \"key=value\""
```

Comment:
Move this check to that function... may you can combine the check `[.]+=[.]+` and have a common error message that it should be in that format.

Also remove the TODO comment after you DO :-)